code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -371,14 +371,14 @@
 		return
 	if(vine_cooldown <= world.time)
 		user.visible_message("<span class='notice'>[user] stabs [src] into the ground.</span>", "<span class='nicegreen'>You stab your [src] into the ground.</span>")
-		for(var/obj/structure/alien/weeds/F0442/F in range(0, get_turf(user)))
+		for(var/obj/structure/alien/weeds/apple_vine/F in range(0, get_turf(user)))
```

Comment:
range 0 is its own turf...

Reply:
I mean yeah, it checks if the person is on top of a vine. Should I make it so it gets the turf of the holder and then checks the contents of that?

Reply:
The result would be the same either way, I suppose.